GitUI/CommandsDialogs/SettingsDialog/Pages/ColorsSettingsPage.cs

```diff
@@ -120,6 +133,12 @@ private void ChkUseSystemVisualStyle_CheckedChanged(object sender, EventArgs e)
             EndUpdateThemeSettings();
         }
 
+        private void ChkColorblind_CheckedChanged(object sender, EventArgs e)
+        {
+            BeginUpdateThemeSettings();
+            EndUpdateThemeSettings();
```

Comment:
This looks strange. Any reason we can't have a single method?

Reply:
ok